plugin/evm/block_verification.go

```diff
@@ -180,18 +180,10 @@ func (v blockValidator) SyntacticVerify(b *Block, rules params.Rules) error {
 
 	// If we are in ApricotPhase4, ensure that ExtDataGasUsed is populated correctly.
 	if rules.IsApricotPhase4 {
```

Comment:
I think we're removing checks from previous upgrades (ap4). Since ap4 is far behind now we should not see any blocks prior ap5, but just wanted to point out.

Reply:
What checks do you think this is removing?

Reply:
ah ok my bad it seems we handle all these cases in switch case below.